reporting-samples/graphql-sample/DataDictionary/CAR_DD.md

```diff
@@ -0,0 +1,63 @@
+# CAR Data Dictionary
+
+| Field Name                                   | Field Type | Description                                                                                                                                                                                             | Is Aggregation allowed? | Is GroupBy allowed? | Is Filter allowed?<br/>(use 'extFilter' argument for all the fields) |
```

Comment:
The last column has the argument of the filter so the heading needs to be fixed

Reply:
changed the heading to "Is Filter allowed?"